src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/controller/OrchestrationSessionsController.kt

```diff
@@ -77,6 +79,47 @@ class OrchestrationSessionsController(private val visitSchedulerService: VisitSc
   ): List<VisitSessionDto>? =
     visitSchedulerService.getVisitSessions(prisonCode, prisonerId, min, max)
 
+  @PreAuthorize("hasRole('VISIT_SCHEDULER')")
+  @GetMapping("/visit-sessions/available")
+  @Operation(
+    summary = "Returns only visit sessions for a prisoner which are within the reservable time period",
```

Comment:
nice to use the word available..

Reply:
Updated - the description is now - "Returns only available visit sessions for a specified prisoner by restriction and within the reservable time period" - same as visit-scheduler